pom.xml

```diff
@@ -65,8 +65,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.6.1</version>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>1.7</source>
+          <target>1.7</target>
```

Comment:
> still need Java 6 support at $JOB.

Since the official Extend Support for Java 6 is quite expensive, most $projects were forced to update :)